services/movie.py

```diff
@@ -1,11 +1,15 @@
+from typing import List
+import init_django_orm  # noqa: F401
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+    genres_ids: List[int] = None,
+    actors_ids: List[int] = None,
+    title: str = None,
```

Comment:
It accepts Optional[str] and Optional[List[int]]
Because as default they accepts None
Fix in all places